src/functions-and-arrays.js

```diff
@@ -52,14 +175,42 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(wordsUnique) {
+  let uniqueArray = [];
+
+  if (wordsUnique.length === 0) {
+    return null
+  }
+
+  for (let i = 0; i < wordsUnique.length; i++) {
+    if (!uniqueArray.includes(wordsUnique[i])) {
+      uniqueArray.push(wordsUnique[i])
+    }
+  }
+
+  return uniqueArray
+}
```

Comment:
Bien resuelto. Nada más comenzar no se espera que encontréis una solución compleja a esta kata pero la forma más elegante (puedes investigarlo) de resolver este problema sería a través de un Set: https://developer.mozilla.org/es/docs/Web/JavaScript/Reference/Global_Objects/Set

Un Set es una estructura de datos que alberga elementos ÚNICOS no repetidos.

Reply:
A ver cuándo le puedo echar un ojo al Set!